beacon-chain/db/filesystem/test_utils.go

```diff
@@ -0,0 +1,83 @@
+package filesystem
+
+import (
+	"testing"
+
+	ckzg4844 "github.com/ethereum/c-kzg-4844/v2/bindings/go"
+	fieldparams "github.com/prysmaticlabs/prysm/v5/config/fieldparams"
+	"github.com/prysmaticlabs/prysm/v5/config/params"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/blocks"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+	ethpb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+)
+
+type (
+	DataColumnParams struct {
+		Slot           primitives.Slot
+		ColumnIndex    uint64
+		KzgCommitments [][]byte
+		DataColumn     []byte // A whole data cell will be filled with the content of one item of this slice.
+	}
+
+	DataColumnsParamsByRoot map[[fieldparams.RootLength]byte][]DataColumnParams
+)
+
+func CreateTestVerifiedRoDataColumnSidecars(t *testing.T, dataColumnParamsByBlockRoot DataColumnsParamsByRoot) ([]blocks.RODataColumn, []blocks.VerifiedRODataColumn) {
```

Comment:
For blobs I wound up moving this sort of functionality into the `verification` package to preserve the condition that other packages do not directly create the `VerifiedRO*` types. I see this requires a `*testing.T` so that does provide some good safety guarantees, but could this file move entirely into the verification package?

https://github.com/prysmaticlabs/prysm/blob/develop/beacon-chain/verification/filesystem.go#L9
https://github.com/prysmaticlabs/prysm/blob/develop/beacon-chain/verification/fake.go#L11

Reply:
Fixed in https://github.com/prysmaticlabs/prysm/pull/15061/commits/30ee3a72257cacd540448b03138605a899753cb6.